site/website/blog/2020-07-16-designing-evergreen-cache-cdc.md

```diff
@@ -277,12 +281,17 @@ pipeline.readFrom(source)                                       //1
 ```
```

Comment:
We have a specialized CDC sink. I think the above code doesn't handle deleted persons. I'm not sure what `r.value.toObject()` returns in that case, but since it is marked as `@Nonnull`, i guess it's the deleted person. But we don't remove that person from the target map, we `put` it there.